samcli/lib/build/build_graph.py

```diff
@@ -351,7 +351,7 @@ def __init__(
     def __str__(self) -> str:
         return (
             f"LayerBuildDefinition({self.name}, {self.codeuri}, {self.source_md5}, {self.uuid}, "
-            f"{self.build_method}, {self.compatible_runtimes}, {self.env_vars}, {self.layer.name})"
+            f"{self.build_method}, {self.compatible_runtimes}, {self.env_vars})"
```

Comment:
This change comes from this commit, that's already merged into `aws:develop`: https://github.com/aws/aws-sam-cli/commit/199ded1901964ed17023e9d5ed5682ff77ae3dfd I'm not sure why it appears here